doc/src/compute_rattlers.rst

```diff
@@ -0,0 +1,77 @@
+.. index:: compute rattlers
+
+compute rattlers command
```

Comment:
Computes that produce per-atom data should have a name like
compute rattler/atom, i.e. suffixed by atom

Reply:
Changed. 

Is there a preference for naming fixes in a similar manner? The fix I added, nonaffine displacement, is kind of an odd duck since it basically acts like a compute (doesn't affect dynamics, calculates per-atom quantities) but I had to construct it as a fix since it needs to operate every timestep.